src/zcash/IncrementalMerkleTree.cpp

```diff
@@ -35,6 +35,392 @@ PedersenHash PedersenHash::uncommitted() {
+        0x7d, 0x2d, 0xa0, 0xe9, 0x0d, 0x06, 0x09, 0x29,
+    }),
+    uint256(std::vector<unsigned char>{
+        0xaa, 0x6e, 0x70, 0xa9, 0x1e, 0xbc, 0x54, 0xee,
+        0xfc, 0xe5, 0xff, 0xd5, 0xb6, 0x75, 0xda, 0xf3,
+        0xf1, 0xd9, 0x40, 0xa8, 0x45, 0x1f, 0xcb, 0x01,
+        0x08, 0x1f, 0xa9, 0xd4, 0xf2, 0x62, 0x43, 0x6f,
+    }),
+    uint256(std::vector<unsigned char>{
+        0xd7, 0x70, 0x38, 0xbf, 0x67, 0xe6, 0x31, 0x75,
+        0x29, 0x40, 0x23, 0x12, 0x51, 0xd7, 0xfe, 0x85,
+        0xaf, 0x52, 0xdb, 0xdd, 0x6a, 0xab, 0x37, 0xc7,
+        0xa5, 0xec, 0x32, 0xb6, 0x5f, 0xe6, 0xde, 0x03,
+    }),
+    uint256(std::vector<unsigned char>{
+        0xd2, 0x27, 0xa1, 0x7a, 0x7e, 0x0c, 0xf9, 0x6d,
+        0xce, 0xdd, 0x9f, 0xc7, 0xbc, 0xe4, 0x3c, 0x6c,
+        0x1d, 0x66, 0xba, 0xdd, 0x75, 0x43, 0xa8, 0x87,
+        0xc8, 0x65, 0x6c, 0x54, 0x7e, 0xcf, 0xb2, 0x4f,
+    }),
+    uint256(std::vector<unsigned char>{
+        0x70, 0xe8, 0xa5, 0x21, 0x95, 0x15, 0x83, 0xe5,
+        0x3f, 0xc0, 0x58, 0x5c, 0x70, 0x7e, 0xce, 0xda,
+        0x89, 0xb7, 0xa7, 0xd1, 0xaf, 0x41, 0xd1, 0xa0,
+        0x15, 0xd7, 0x97, 0xfa, 0x76, 0xc0, 0xf5, 0x69,
+    }),
+    uint256(std::vector<unsigned char>{
+        0xe4, 0x85, 0xa9, 0x68, 0x55, 0xe8, 0x72, 0xfc,
+        0x50, 0x90, 0x15, 0x0e, 0x2c, 0xd2, 0x4e, 0x10,
+        0x59, 0x1d, 0x35, 0x16, 0x6e, 0xb0, 0xeb, 0x30,
+        0xfc, 0xdf, 0xac, 0x93, 0xb0, 0x1d, 0x28, 0x1c,
+    }),
+    uint256(std::vector<unsigned char>{
+        0xe4, 0xa1, 0x9f, 0xeb, 0xdf, 0x2a, 0x86, 0x89,
+        0x6e, 0x41, 0xf2, 0xce, 0xdc, 0xf2, 0xae, 0x58,
+        0x46, 0x71, 0x80, 0x2e, 0x6a, 0x46, 0x7e, 0x84,
+        0x39, 0xca, 0xb5, 0xd6, 0x18, 0x43, 0x41, 0x6b,
+    }),
+    uint256(std::vector<unsigned char>{
+        0xe9, 0x27, 0x83, 0x88, 0x47, 0x80, 0x6a, 0x43,
+        0xbd, 0x6c, 0x60, 0x88, 0xe3, 0x9f, 0x65, 0xb8,
+        0xb3, 0xe5, 0x8b, 0x2d, 0xb5, 0xf7, 0xad, 0x56,
+        0x43, 0xd9, 0x1e, 0x06, 0x59, 0xa2, 0x8a, 0x2a,
+    }),
+    uint256(std::vector<unsigned char>{
+        0x0b, 0xd3, 0xa8, 0x18, 0xe8, 0x3f, 0x9c, 0xd2,
+        0xff, 0x4f, 0x62, 0x01, 0x1a, 0x51, 0x01, 0x76,
+        0xac, 0x32, 0xf5, 0x44, 0x8e, 0x6e, 0x15, 0x45,
+        0x15, 0x04, 0x3c, 0x59, 0x26, 0xd5, 0x1c, 0x6f,
+    }),
+    uint256(std::vector<unsigned char>{
+        0xce, 0x41, 0x34, 0x45, 0xe0, 0x37, 0x90, 0x49,
+        0x8f, 0xe7, 0x2d, 0x8e, 0x01, 0x91, 0x5e, 0x7f,
+        0xf1, 0x20, 0xae, 0x35, 0xb3, 0xb5, 0x90, 0xd2,
+        0x1b, 0x7f, 0x74, 0xde, 0xe1, 0x83, 0x0f, 0x0d,
+    }),
+    uint256(std::vector<unsigned char>{
+        0x60, 0x0e, 0x6f, 0x93, 0xe7, 0x3d, 0x7a, 0xbd,
+        0x4e, 0xe0, 0xa6, 0x5c, 0xb1, 0xb1, 0x9a, 0xa3,
+        0xec, 0xc5, 0x25, 0x68, 0x9d, 0xbf, 0x17, 0x77,
+        0x96, 0x58, 0x74, 0x1b, 0x95, 0xc1, 0x5a, 0x55,
+    }),
+};
+
+PedersenHash PedersenHash::EmptyRoot(size_t depth) {
+    assert(depth <= sizeof(pedersen_empty_roots)/sizeof(pedersen_empty_roots[0]));
```

Comment:
Why not `depth < pedersen_empty_roots.size()`?

Reply:
```suggestion
    assert(depth < pedersen_empty_roots.size());
```

Reply:
```suggestion
```

Reply:
Better to remove the assert?